packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/components/Stages/Stage/Stage.js

```diff
@@ -61,6 +61,39 @@ const DeleteMenuItem = styled(MenuItem)`
   color: ${({ theme }) => theme.colors.danger600};
 `;
 
+// Removing the font-size from the child-span aligns the
+// more icon vertically
+const ContextMenuTrigger = styled(Menu.Trigger)`
+  :hover,
+  :focus {
+    background-color: ${({ theme }) => theme.colors.neutral100};
+  }
+
+  > span {
+    font-size: 0;
+  }
```

Comment:
I think this is an issue with the DS... I don't think it makes sense to always have a typography component inside `Button` – something to discuss later though.